packages/base-styles/_variables.scss

```diff
@@ -83,7 +83,7 @@ $widget-area-width: 700px;
 
 $block-toolbar-height: $grid-unit-60;
 $border-width: 1px;
-$border-width-focus: 2px; // This exists as a fallback, and is ideally overridden by var(--wp-admin-border-width-focus) unless in some SASS math cases.
+$border-width-focus-fallback: 2px; // This exists as a fallback, and is ideally overridden by var(--wp-admin-border-width-focus) unless in some SASS math cases.
```

Comment:
I guess there's a technical bent to that question that I can't respond to. However if it's up for discussion I would strongly recommend against that, as these are meant to be UI only styling, not something that's meant to be leveraged by 3rd parties. Many variables have been created by accident and retired, many others have been renamed. I agree it's something to avoid when possible, but also just a fact of how we've cleaned things up. So I'd keep this private.

Reply:
I ran a quick search of WP directory:

- [no matches on third party Themes](https://wpdirectory.net/search/01GZ3SQ7MRZC5PSZ4RFS3B27HR)
- [a couple of matches on third-party plugins](https://wpdirectory.net/search/01GZ3SNZ9RW87Z2VRVGD9A8SDW) (although that's because they are bundling the Gutenberg source code)

So, in that sense this variable renaming is safe and I think that we can merge this PR.

--------

> However if it's up for discussion I would strongly recommend against that, as these are meant to be UI only styling, not something that's meant to be leveraged by 3rd parties. Many variables have been created by accident and retired, many others have been renamed. I agree it's something to avoid when possible, but also just a fact of how we've cleaned things up. So I'd keep this private.

That sounds totally reasonable to me, although I've already been part of situations in the past where similar points were made against my changes 🤷‍♂️  I guess the bulk of the argument would be to understand whether those SCSS variables: the package's README states "Base SCSS utilities and variables for WordPress.", and therefore I see how it could be assumed that those variables are indeed part of the "public APIs" exposed by the package.